src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -65,7 +65,7 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
     // Values we want all provider configs to by default contain
     configFormData.append('enterprise_customer_uuid', enterpriseId);
     configFormData.append('enabled', true);
-    configFormData.append('debug_mode', true);
+    configFormData.append('debug_mode', providerConfig?.debug_mode || false);
```

Comment:
This seems like the right way to go.

We should do the same for the automatic refresh setting on line 73 -- `providerConfig?.automatic_refresh_enabled || false`

Reply:
yup, happy to. Do we want it to default to true or false?

Reply:
I think it should default to false